lib/private/docker.ts

```diff
@@ -284,7 +294,7 @@ export class DockerFactory {
         return;
       }
 
-      await docker.login(options.ecr);
+      await docker.login(options.ecr, options.eventEmitter);
```

Comment:
Now I *know* we passed `options.eventEmitter` to `new Docker()`... so presumably `docker.login()` could do `this.eventEmitter` inside the method to get access to it.

Why is it necessary to pass it again here?

Reply:
it's not! Good catch